monitoring-micrometrics-grafana-prometheus/src/main/java/org/apache/camel/example/spring/boot/monitoring/Application.java

```diff
@@ -19,9 +19,10 @@
 import io.micrometer.core.instrument.Clock;
 import io.micrometer.prometheusmetrics.PrometheusConfig;
 import io.micrometer.prometheusmetrics.PrometheusMeterRegistry;
-import io.prometheus.client.CollectorRegistry;
-import io.prometheus.jmx.BuildInfoCollector;
+//import io.prometheus.client.CollectorRegistry;
```

Comment:
can this comment be removed?

Reply:
Someone needs to fix this example - I want to be obvious about what was there so it's easier to fix